pkgs/servers/deconz/default.nix

```
@@ -0,0 +1,83 @@
+{ stdenv
+, lib
+, fetchurl
+, wrapQtAppsHook
+, dpkg
+, autoPatchelfHook
+, qtserialport
+, qtwebsockets
+, openssl
+, libredirect
+, makeWrapper
+, gzip
+, gnutar
+}:
+
+stdenv.mkDerivation rec {
+  name = "deconz-${version}";
```

It looks like code from 5 years ago.

```suggestion
  pname = "deconz";
```

---

Hehe, that's because it is! Thanks for spotting.